src/accounts/accountsService.js

```diff
@@ -88,7 +88,7 @@ const resetPassword = async ({ token, newPassword }) => {
 
   if (!userToken.token === token) throw new BadRequestError('Invalid token');
 
-  const isExpired = userToken.expiryTime.getTime() <= Date.now();
+  const isExpired = userToken.expiresAt.getTime() <= Date.now();
```

Comment:
The changes introduced in [this commit](https://github.com/AltCamp/altcampv1-backend/pull/189/commits/43611c7bf7e174cd1a1df0884113ad8891aa9546) from #189 eliminate the need for this check.

The database deletes the token after the expiry date has been set. 
Please can we introduce this change in another PR? We can use this one to update the docs 🙏🏾